syntax = "proto3";

package proto;

message NodeConfig {
  message NodeAddress {
    string ip_address = 1;
    string port = 2;
  }
  repeated NodeAddress nodes = 1;
}
